specs/core/0_beacon-chain.md

```diff
@@ -483,9 +478,9 @@ The beacon chain fork choice rule is a hybrid that combines justification and fi
 
 ```python
 def lmd_ghost(store, start):
-    validators = start.state.validators
-    active_validators = [validators[i] for i in
-                         get_active_validator_indices(validators, start.slot)]
+    validator_set = start.state.validator_set
```

Comment:
This is the first renaming I don't immediately agree with. Why `validators` -> `validator_set`?